kafka-ui-react-app/src/components/Topics/List/__tests__/List.spec.tsx

```diff
@@ -100,13 +101,35 @@ describe('List', () => {
       expect(setTopicsSearch).toHaveBeenCalledWith(query);
     });
 
+    it('show internal toggle state should be true if user has not used it yet', () => {
+      const toggle = component.find('input[name="ShowInternalTopics"]');
```

Comment:
if this one repeats , we should put it in variable and re-use it in our test **it** scope

Reply:
Yes. It repeats, but because we reassign the component variable in many places (not here), we'll have to reassign the toggle, otherwise, the tests will fail. So, I decided to store the name ('input[name="ShowInternalTopics"]') in a variable and use it everywhere to find the ShowInternalTopics wrapper inside 'component' and prevent repetition.

Reply:
Okay good for me I guess.